lib/csv/writer.rb

```diff
@@ -14,6 +14,7 @@ def initialize(output, options)
       @output = output
       @options = options
       @lineno = 0
+      @fields_converter = nil
```

Comment:
Ah, I need to confirm why this is needed. (TODO)

Reply:
In `CSV::Writer#initialize`:

```ruby
      if @options[:write_headers] and @headers
        self << @headers
      end
```

and `CSV::Writer#<<` refers `@fields_converter`.

```ruby
      row = @fields_converter.convert(row, nil, lineno) if @fields_converter
```

Reply:
I wasn't sure if the converter can be applied to headers too.

Reply:
Thanks. It should not be applied to headers.
So this is OK.